recipes/stereo-pipeline/recipe.yaml

```diff
@@ -0,0 +1,154 @@
+      target_directory: StereoPipeline
+    
+    # Submodules
+    
+    - url: https://github.com/NeoGeographyToolkit/StereoPipeline/releases/download/${{ version }}/MultiView.tar.gz
+      sha256: d69d34adc263dceb023bbfd0910c3ab0e69a3dc67292c046860e924e21a2658a
+      target_directory: MultiView
+
+    - url: https://github.com/NeoGeographyToolkit/StereoPipeline/releases/download/geoid1.0/geoids.tgz
+      sha256: 2cffdac6efbfa47a94e8adb650489a267de72770f06656df35f37a531d999f5d
+      target_directory: geoids
+    
+    - url: https://github.com/NeoGeographyToolkit/libnabo/archive/refs/tags/3.5.0.zip
+      sha256: 4b260a5ff3fc6fff56ff500d2b14c1cc4136ac58a3ed306e78dee8769261ae3c
+      target_directory: libnabo
+
+    - url: https://github.com/NeoGeographyToolkit/libpointmatcher/archive/refs/tags/3.5.0.zip
+      sha256: 75f58c416d98691f0c88cd5eb712dfaeb2fb478d1c21a14065ed6c17264310f8
+      target_directory: libpointmatcher
+      
+    - url: https://github.com/NeoGeographyToolkit/FastGlobalRegistration/archive/refs/tags/3.5.0.zip
+      sha256: 3bd48acc05af13fc5dfbf4ca397f9ffa28fd53c83a28346358dff0726e4d2996
+      target_directory: FastGlobalRegistration
+      
+    - url: https://github.com/NeoGeographyToolkit/s2p/releases/download/3.5.0/s2p.tar.gz
+      sha256: 8752edac97f9596266d2712123ea1d14e44e7c201c8ea4062223708886ae5237
+      target_directory: s2p
+      
+    - url: https://github.com/NeoGeographyToolkit/libelas/archive/refs/tags/3.5.0.zip
+      sha256: d160bbfd020aefac4782a5b269d9c0f8d6af5f43cb8951d27204d2068315521e
+      target_directory: libelas
+
+build:
+  number: 0
+  skip: win
+  script: build.sh
+
+requirements:
+  build:
+    - ${{ compiler("c") }}
+    - ${{ compiler("cxx") }}
+    - ${{ compiler("fortran") }}
+    - ${{ stdlib("c") }}
+    - cmake >=3.15
+    - make
+    
+  host:
+    - ale
+    - csm
+    - ceres-solver
+    - eigen
+    - flann
+    - fftw
+    - geotiff
+    - gflags
+    - glog
+    - libboost-devel
+    - libgdal
+    - libblas * *openblas
+    - libopencv
+    - libjpeg-turbo
+    - libpng
+    - libpdal
+    - libtiff
+    - libxml2
+    - llvm-openmp
+    - mesalib
+    - parallel
+    - perl
+    - pcl
+    - protobuf
+    - qt-main
+    - qwt
+    - qhull
+    - rapidjson
+    - rocksdb
+    - tbb
+    - tbb-devel
+    - suitesparse
+    - superlu
+    - usgscsm
+    - visionworkbench
+    - vlfeat
+    - xerces-c
+
+  run:
+    - ale
+    - csm
+    - ceres-solver
+    - eigen
+    - flann
+    - fftw
+    - geotiff
+    - gflags
+    - glog
+    - libboost-devel
+    - libgdal
+    - libblas * *openblas
+    - libopencv
+    - libjpeg-turbo
+    - libpng
+    - libpdal
+    - libtiff
+    - libxml2
+    - llvm-openmp
+    - mesalib
+    - parallel
+    - pcl
+    - protobuf
+    - qt-main
+    - qwt
+    - qhull
+    - rapidjson
+    - rocksdb
+    - tbb
+    - tbb-devel
+    - suitesparse
+    - superlu
+    - usgscsm
+    - visionworkbench
+    - vlfeat
+    - xerces-c
```

Comment:
Most of these have `run_exports` and should not be listed in `run`. See the cleanup done in visionworkbench.

Reply:
Addressed.

Reply:
The build failed when I took most of these out. Now I keep only a handful.